CONTRIBUTING.md

```diff
@@ -37,193 +37,210 @@ To add a new page to the website, [create a new markdown file with YAML frontmat
 
 ```yaml
 ---
+# Name of the product (mandatory).
 title: Timeturner
-# Possible values are os,db,app,lang,framework,device,service,server-app
+
+# Category of the product (mandatory).
+# Possible values are os,db,app,lang,framework,device,service,server-app.
 # If you add a new value, please mention it on the PR Description. Some rough guidelines:
-# os is operating systems (and similar projects)
-# db is databases, os is operating systems, lang is programming languages
-# app is end-user applications
-# lang is programming languages
-# framework is used for application libraries, SDKs, frameworks etc
-# device is physical devices
-# service is managed service offerings (SaaS/PaaS etc)
-# server-app are applications usually installed on the server-side
+# - os is for operating systems (and similar projects)
+# - db is for databases,
+# - app is for end-user applications,
+# - lang is for programming languages,
+# - framework is for application libraries, SDKs, frameworks...,
+# - device is for physical devices,
+# - service is for managed service offerings (SaaS/PaaS...),
+# - server-app is for applications usually installed on the server-side.
 category: os
 
-# Template to be used to generate a link for the release
-# __RELEASE_CYCLE__ will be replaced by the value of releaseCycle
-# __LATEST__ will be replaced by the value of latest
-# __CODENAME__ will be replaced by the optional codename
+# Simple Icons (https://simpleicons.org/) icon slug (mandatory).
+# If the icon is not available on Simple Icons, set it to `NA`.
+# As an example, https://simpleicons.org/?q=codemagic links to https://simpleicons.org/icons/codemagic.svg,
+# so the slug is `opensuse` (the SVG filename without extension).
+# A list of all slugs is also available on https://github.com/simple-icons/simple-icons/blob/develop/slugs.md.
```

Comment:
could we make a space between dot and md part at the end of line ? right now it looks a bit wrong